src/server.ts

```diff
@@ -1,15 +1,41 @@
-//import * as express from "express";
-//import express from "express";
 import express, { Express } from "express";
+import axios from 'axios';
+import {API_KEY} from "./info";
  
 const app = express();
 const port = 8000;
  
 app.use(express.json());
+
+// Part 2
+
+// 1. We can retrieve the list of rovers from
+// https://api.nasa.gov/mars-photos/api/v1/rovers?api_key=API_KEY
+
+// 2 create GET endpoint for the rovers
+
 const router = express.Router();
-router.get('/test', (req, res) => res.send('Hello world !'));
+
+router.get('/rovers', function (req, res) {
+
+  console.log("Getting rovers ...");
+  const URL = `https://api.nasa.gov/mars-photos/api/v1/rovers?api_key=${API_KEY}`;
+
+  axios.get(URL)
+  .then((response) => {
+    let result = response.data;
```

Comment:
We talked a little about let and const yesterday during the typescript presentation, but as you've already seen :)), I don't know how to use them quite right. I will change this and make the other suggested changes as well in a few minutes and I will let you know!

Reply:
A good rule of thumb is that if you ever have to use a `let`, you can _probably_ write your code in a different way using a `const` and it will work out better.

Reply:
Thank you very much for the entire feedback!